### PR TITLE
add f30 docker to run avocado

### PR DESCRIPTION
**Pull request by amoskong**
Huginn had been upgraded from Fedora 30 to Fedora 32.
All python2 packages had been removed.
(2019.1) artifact-test jobs of ubuntu/debian failed for lack of Avocado.

This patch switched Dockerfile from CentOS7 to Fedora30.
The test job already passed.

Signed-off-by: Amos Kong <amos@scylladb.com>

### CODE REVIEW

**Comment by fruch** [comment]
@amoskong are you running with this docker ? or it's just for testing or suggesting a fix ?

File: requirements-python.txt [hunk 1]
```diff
@@ -1,18 +1,29 @@
 ## Avocado functional requirements
 ## SSH lib (avocado.utils.remote)
-fabric==2.4.0
+fabric==fabric==1.11.3
```
Comment (fruch) [inline requirements-python.txt:3]: this seems wrong....

**Comment by amoskong** [comment]
> @amoskong are you running with this docker ? or it's just for testing or suggesting a fix ?

Currently artifact-tests of 2019.1 (only ubuntu & debian) failed.
I tested on huginn with this docker image, it solved the problem.

When we review and merge this PR. I will sync with release engineer again with the change.
I already posted my idea in releg maillist.

File: requirements-python.txt (hunk 1 above)
Comment (amoskong) [reply requirements-python.txt:3]: fixed

**Review by fruch — APPROVED** [review]
LGTM

**Pull request merged**